safe-global/safe-wallet-web
# Feat(sidebar): sort safes by name and recently used [SW-307]

## What it solves

Resolves: https://www.notion.so/safe-global/Add-ability-to-sort-safes-list-11f8180fe5738031ba4fdda3245b0508?pvs=4

## How this PR fixes it
- adds name and last visited timestamp to safe list items and multi chain account items
- uses these to reorder the safes list depending on what's chosen in the sort by dropdown.

## How to test it
- Open the sidebar or accounts page.
- The safes should be ordered with the most recently visited safes appearing first by default. Multichain safes should also have their sub safes ordered by last visited.
- Change the order by preference to name and the safes should be ordered by name in ascending alphabetical order with nameless safes appearing at the end.

## Screenshots
![image](https://github.com/user-attachments/assets/e8c1c1ab-1187-4297-96d1-866239fc4f59)
![image](https://github.com/user-attachments/assets/e23f1aa1-9726-4e9d-a6f9-7508f9bc53f8)
![image](https://github.com/user-attachments/assets/7333dac5-4122-4174-a91d-f2c6052908c3)


## Checklist
